include/rocksdb/db.h

```diff
@@ -1671,6 +1667,30 @@ class DB {
 #endif  // !ROCKSDB_LITE
 };
 
+// Overloaded operators for enum class SizeApproximationFlags.
+inline DB::SizeApproximationFlags operator&(DB::SizeApproximationFlags lhs,
+                                            DB::SizeApproximationFlags rhs) {
+  return static_cast<DB::SizeApproximationFlags>((uint8_t)lhs & (uint8_t)rhs);
```

Comment:
Let's use `static_cast` in favor of C-style type conversion for `lhs` and `rhs`.